CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+  - CIP-108: Off-chain metadata for proposals or related governance documents.
+
+**Note:** The technical details of each CIP (hashing, labeling, datum layouts) are **not** restated here. Implementers must consult each CIP’s original documentation.
+
+### 2. Identifiers (DRep & Proposal)
+
+1. **DRep ID**
+
+   - A **28-byte (56-hex)** blake2b-224 hash of the stake key or script credential per CIP-1694.
+
+2. **Proposal ID**
+   - A string usually of the form `"txHash + index"`, as outlined by CIP-1694.
+
+All references to DReps or proposals in these NFTs **must** use these official IDs, ensuring consistency with CIP-1694 governance tooling.
+
+### 3. On-Chain vs. Off-Chain Metadata
+
+**On-Chain**
+
+- The CIP-68 datum holds minimal but essential governance fields—IDs, short texts, timestamps.
+- Self-reported data (e.g., a DRep’s focus areas or a quick rationale) is possible but should remain concise to avoid high on-chain storage costs.
+
+**Off-Chain**
+
+- Large or frequently updated content (e.g., multi-paragraph mission statements, proposal PDFs, rich media) is stored off-ledger.
+- References take the form of a **URL + blake2b-256 hash** (per CIP-119 or CIP-108), allowing client apps to verify integrity while avoiding ledger bloat.
+
+### 4. Governance NFT Types
+
+All governance NFTs proposed here share a **new CIP-67 label** (placeholder `(1694)`). Example asset names:
+
+- `"(1694)drepCredential-<dRepAlias>"`
+- `"(1694)ballotNote-<proposalId>"`
+- `"(1694)endorsement-<endorserKey>-<dRepId>"`
+
+#### 4.1 DRep Credential
+
+**Minted by:** The DRep or an authorized delegate.  
+**Purpose:**
+
+- Captures key DRep info on-chain (e.g., `dRepId`, name, roles).
+- Optionally anchors extended data via CIP-119.
+- Reflects the “DRep Credential Form” mockup fields.
+
+**Core Field:**
+
+- `dRepId` (CIP-1694 hash)
+
+**Recommended Optional Fields (Self-Reported):**
+
+- **Title/Role**: Short descriptor (e.g., “DeFi Developer”).
+- **Name (or Alias)**, **Profile Image URL**, **Website**, **Mission/Goals**.
+- **Primary Focus Areas** (e.g., DeFi, Governance, Education).
+- **Roles** (SPO, Dev/Builder, Community Manager, etc.).
+- **Sectors** (NFTs, Identity, Gaming, DAO Tools).
+- **Social Links** (Twitter handle, Discord handle).
+- **Off-Chain Anchor** (`cip119AnchorUrl`, `cip119AnchorHash`) for extended profiles.
+
+**Minimal JSON Example:**
+
+```jsonc
+{
+  "version": 1,
+  "dRepId": "b2f0...20fb",
+  "titleOrRole": "DeFi Developer",
+  "nameOrAlias": "Alice The Builder",
+  "missionOrGoals": "Promoting transparent DeFi governance on Cardano",
+  "primaryFocusAreas": ["DeFi", "Governance"],
+  "roles": ["Dev/Builder", "Community Manager"],
+  "website": "https://mydrepwebsite.com",
+  "twitterHandle": "@aliceBuilder",
+  "cip119AnchorUrl": "ipfs://bafy123...",
+  "cip119AnchorHash": "blake2b256:abc123..."
+}
+```
+
+#### 4.2 Ballot Note
+
+**Minted by:** A DRep publishing a stance on a proposal.  
+**Purpose:**
```

Comment:
what happens if a DRep's Ballot Note doesn't match their vote?

It would be worth describing the exact problem that 'Ballot Note' solves